src/embind/emval.js

```diff
@@ -413,9 +406,7 @@ var LibraryEmVal = {
           types[i + 1].deleteObject(argN[i]);
         }
       }
-      if (!retType.isVoid) {
```

Comment:
Maybe we should leave this check so the DYNAMIC_EXECUTION=0/1 code remains more symmetric. e.g. They both don't call toWireType when the return type is null.

Reply:
toWireType is no-op for void anyway. For dynamic execution the check makes sense because it avoids emitting `toWireType` into the generated function altogether, but for non-dynamic case it's easier / less code to just call the method.

Reply:
Also FWIW I have a more substantial rewrite of this method upcoming anyway.